twiliotutorial/views.py

```diff
@@ -55,3 +67,74 @@ def handle_digits(self, digits):
             response.hangup()
 
         return response
+
+
+class BeerFact(View):
+
+    def post(self, request):
+        response = voice_response.VoiceResponse()
+
+
+        beer = Beer()
+        mug = beer.get_beer_fact()
+        name = mug['name']
+        logging.debug("Beer info: %s".format(beer))
+        response.say("Hello! I am going to drop a dank beer on you.", voice=settings.VOICE)
+        gather = voice_response.Gather(
+            action="/beertext?beerid=" + mug['id'],
+            action_on_empty_result=False,
+            timeout=200,
+            numDigits=1,
+            finishOnKey='')
+        gather.say(f"Our beer today is {name}", voice=settings.VOICE)
+
+        if 'abv' in mug.keys():
+            abv = mug['abv']
+            gather.say(f"Coming in at {abv} percent.", voice=settings.VOICE)
+
+        if 'style' in mug.keys():
+            if 'description' in mug['style'].keys():
+                description = mug['style']['description']
+                gather.say(f"{description}", voice=settings.VOICE)
+        response.append(gather)
+        response.say("Wow! You made it to the end. Be excellent to each other.", voice=settings.VOICE)
+        response.hangup()
+
+        response = HttpResponse(response.to_xml())
```

Comment:
We are setting the content body to xml. What's the mime type on the reponse? JSON? text/plain?

Reply:
Seems like the response is html which should work for xml, Twilio doesnt mind.